### PR TITLE
[DOCFIX] Update documentation for ls command

### PR DESCRIPTION
**Pull request by peisun1115**

### CODE REVIEW

File: docs/cn/Command-Line-Interface.md [hunk 1]
```diff
@@ -199,6 +199,9 @@ fs命令中的所有“路径”都应该以以下开头：
 
 加上`-R`选项可以递归的列出输入路径下的所有子文件和子文件夹，并列出从输入路径开始的所有子树。
 
+加上`-f`选项可以强制访问本地文件系统, 创建子文件在alluxio系统中的镜像。该命令只创建元数据, 例如文件名及文件大小，而不会传输数据。如果不加
```
Comment (apc999) [inline docs/cn/Command-Line-Interface.md]: NIT: alluxio -> Alluxio

Comment (peisun1115) [inline docs/cn/Command-Line-Interface.md]: done

**Comment by calvinjia** [comment]
LGTM

**Comment by peisun1115** [comment]
Anyone help to merge it?

**Comment by peisun1115** [comment]
Jenkins, test this please

File: docs/_includes/Command-Line-Interface/ls.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
 ```bash
-$ ./bin/alluxio fs ls /users/alice/
+$ ./bin/alluxio fs mount s3n://data-bucket/ /s3/data
```
Comment (gpang) [inline docs/_includes/Command-Line-Interface/ls.md]: are the arguments backwards here? i thought the first was alluxio path and the second was ufs uri?

**Comment by gpang** [comment]
I had a minor question. @peisun1115 Thanks!

File: docs/_includes/Command-Line-Interface/ls.md (hunk 2 above)
Comment (peisun1115) [inline docs/_includes/Command-Line-Interface/ls.md]: I copied from mount.md :) Fixed this and mount.md

**Comment by peisun1115** [comment]
@gpang Can you take another look? Thanks

**Pull request merged**